examples/with-sentry/components/withSentry.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import Raven from 'raven-js'
+
+function withSentry (Child) {
+  return class WrappedComponent extends React.Component {
+    static getInitialProps (context) {
+      return Child.getInitialProps(context)
+    }
+    constructor (props) {
+      super(props)
+      this.state = {
+        error: null
+      }
+      Raven.config(
+        // change this for your Sentry DSN
+        'https://cb0826e4617043e1bdafcb519df0992b@sentry.io/237733'
+      ).install()
+    }
+
+    componentDidCatch (error, errorInfo) {
+      this.setState({ error })
+      Raven.captureException(error, { extra: errorInfo })
+    }
+
+    render () {
+      return <Child {...this.props} />
```

Comment:
Or if you don't use 2 components pass `error={this.state.error` here

Reply:
Did that instead of 2 components. See related comment.